net.granjow.slowmovideo.yaml

```diff
@@ -0,0 +1,84 @@
+app-id: net.granjow.slowmovideo
+runtime: org.kde.Platform
+runtime-version: '5.15-21.08'
+sdk: org.kde.Sdk
+x-add-extensions:
```

Comment:
`x-add-extensions` ?

Reply:
ah, I played around with disabling ffmpeg. Let me remove this. Thanks.